pkg/controller/servicebindingrequest/retriever.go

```diff
@@ -171,7 +171,7 @@ func (r *Retriever) readAnnotation(key, value string) error {
 	if len(fullpathslice) > 1 {
 		value = value + ":" + fullpathslice[1]
 	}
-	logger.Debug("Value", value)
+	logger.Debug("Value", "%#v", value)
```

Comment:
@baijum you will need to use something like this:

```go
log.WithValues("SectionMap", sectionMap).Debug("<some-interesting-log-message here>")
```
```

Reply:
Fixed 01058a52b00a36a82504b26f8094d5d745e8d475